src/plone/app/theming/browser/controlpanel.pt

```diff
@@ -181,11 +186,12 @@
                                     name="form.button.InvalidateCache">Clear Cache</button>
                             </form>
 
-                            <a href="#" class="plone-btn plone-btn-default pat-plone-modal"
+                            <!-- <a class="plone-btn plone-btn-default pat-plone-modal"
```

Comment:
Is this commented code a WIP? I think we should get rid of commented code in pts.